android/settings.gradle

```diff
@@ -4,4 +4,4 @@ project(':react-native-image-picker').projectDir = new File(rootProject.projectD
 include ':react-native-background-upload'
 project(':react-native-background-upload').projectDir = new File(rootProject.projectDir, '../node_modules/react-native-background-upload/android')
 
-include ':app'
+include ':app'
```

Comment:
intentional?

Reply:
Yes, added by `react-native-git-upgrade`.  That's how new projects have it.  Not sure why though